release.py

```diff
@@ -4,8 +4,44 @@
 import ast
 import subprocess
 import sys
+from optparse import OptionParser
 
 DEBUG = False
+CONFIRM_STEPS = False
+DRY_RUN = False
+
+
+def skip_step():
+    """
+    Asks for user's response whether to run a step. Default is yes.
+    :return: boolean
+    """
+    global CONFIRM_STEPS
```

Comment:
I agree the dry run is pretty awesome. 

Since the `global` qualifier doesn't hurt anything, I'm gonna merge this in.